File: app/src/main/java/com/fundit/model/CategoryResponse.java
package com.fundit.model;

import java.util.ArrayList;

/**
 * Created by prince on 7/19/2017.
 */

public class CategoryResponse extends AppModel {

    ArrayList<AreaItem> data=new ArrayList<>();

    public ArrayList<AreaItem> getData() {
        return data;
    }

    public ArrayList<String> getIDList(){
        ArrayList<String> idList=new ArrayList<>();

        for (int i = 0; i < data.size(); i++) {
            idList.add(data.get(i).getId());
        }

        return idList;
    }

    public ArrayList<String> getNameList(){
        ArrayList<String> nameList=new ArrayList<>();

        for (int i = 0; i < data.size(); i++) {
            nameList.add(data.get(i).getName());
        }

        return nameList;
    }
}

File: app/src/main/java/com/fundit/model/CategoryResponse.kt
package com.fundit.model

/**
 * Created by Nivida new on 17-Jul-17.
 */
class CategoryResponse : AppModel(){

    var data: ArrayList<Category> = ArrayList()

    class Category{
        var id: String = ""
        var name: String = ""
        var parent_id: String = ""
        var image: String = ""
    }
}